student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(_classrom, age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] `classroom` is a necessary argument, please remove the `_`, because it means that the argument is optional 😄